.github/workflows/release-docs.yml

```diff
@@ -1,10 +1,25 @@
 name: publish docs to github pages
 
 on:
+  # Runs on pushes targeting the default branch
   release:
     types: [published]
+
+  # Allows you to run this workflow manually from the Actions tab
   workflow_dispatch:
 
+# Sets permissions of the GITHUB_TOKEN to allow deployment to GitHub Pages
+permissions:
+  contents: read
+  pages: write
+  id-token: write
+
+# Allow only one concurrent deployment, skipping runs queued between the run in-progress and latest queued.
+# However, do NOT cancel in-progress runs as we want to allow these production deployments to complete.
+concurrency:
+  group: "pages"
+  cancel-in-progress: false
```

Comment:
Should this be true so we know we always get the 'latest'? We would only cancel in-progress if there was a new one right?

Reply:
Not sure; this is what is suggested explicitly by the example flows for github pages. I suspect it will be very rare we encounter it.